core/src/trezor/lvglui/scrs/homescreen.py

```
@@ -558,16 +582,156 @@
             self.guide_desc.set_text(_(i18n_keys.APP__TIPS))
-            self.address_desc.set_text(_(i18n_keys.APP__ADDRESS))
             self.nft_desc.set_text(_(i18n_keys.APP__NFT_GALLERY))
             self.backup_desc.set_text(_(i18n_keys.APP__BACK_UP))
             self.scan_desc.set_text(_(i18n_keys.APP__SCAN))
             self.connect_desc.set_text(_(i18n_keys.APP__CONNECT_WALLET))
+            self.my_address_desc.set_text(_(i18n_keys.APP__ADDRESS))
+            self.passkey_desc.set_text(_(i18n_keys.FIDO_FIDO_KEYS_LABEL))
+
+
+class PasskeysManager(Screen):
+    def __init__(self, prev_scr=None):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)